src/services/ui/src/pages/form/chip-form.tsx

```diff
@@ -214,7 +237,16 @@ export const ChipForm = () => {
             <Inputs.Button
               type="button"
               variant="outline"
-              onClick={() => setCancelModalOpen(true)}
+              onClick={() => {
+                setModalContent({
+                  header: "Stop form submission?",
+                  body: "All information you've entered on this form will be lost if you leave this page.",
+                  acceptButtonText: "Yes, leave form",
+                  cancelButtonText: "Return to form",
+                });
+                setModalOnAccept(() => acceptAction);
```

Comment:
I haven't tested yet, but this looks like it needs to be updated to:

```js
setModalOnAccept(acceptAction);
```

Reply:
This works as-intended. To properly test/confirm, change the `() => acceptAction` to `acceptAction` and try the Cancel button on this, or the other, forms.